src/main/java/stsjorbsmod/cards/cull/deckoftrials/Sacrifice.java

```diff
@@ -0,0 +1,52 @@
+package stsjorbsmod.cards.cull.deckoftrials;
+
+import com.megacrit.cardcrawl.actions.common.ApplyPowerAction;
+import com.megacrit.cardcrawl.actions.common.GainEnergyAction;
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import stsjorbsmod.JorbsMod;
+import stsjorbsmod.cards.CustomJorbsModCard;
+import stsjorbsmod.characters.Cull;
+import stsjorbsmod.patches.SelfExertField;
+import stsjorbsmod.powers.SacrificePower;
+
+public class Sacrifice extends CustomJorbsModCard {
+    public static final String ID = JorbsMod.makeID(Sacrifice.class);
+
+    private static final CardRarity RARITY = CardRarity.SPECIAL;
+    private static final CardTarget TARGET = CardTarget.SELF;
+    private static final CardType TYPE = CardType.SKILL;
+    public static final CardColor COLOR = Cull.Enums.CULL_CARD_COLOR;
+
+    private static final int COST = 0;
+    private static final int ENERGY_GAIN = 1;
+    private static final int UPGRADE_ENERGY_GAIN = 1;
+
+    public Sacrifice() {
+        super(ID, COST, TYPE, COLOR, RARITY, TARGET);
+        baseMagicNumber = magicNumber = ENERGY_GAIN;
+
+        this.isEthereal = true;
+        SelfExertField.selfExert.set(this, true);
+    }
+
+    @Override
+    public void use(AbstractPlayer p, AbstractMonster abstractMonster) {
+        addToBot(new ApplyPowerAction(p, p, new SacrificePower(p, magicNumber)));
```

Comment:
I think this should be using `1`, not `magicNumber` (but I appreciate that the power handles the >1 case, I think that's a reasonable choice for handling stacking multiple copies of Sacrifice)